source/wp-content/themes/wporg-news-2021/sass/components/_query-title-banner.scss

```diff
@@ -26,7 +26,10 @@ body.search,
 body.archive:not(.category) {
```

Comment:
I noticed a few pages where `header.query-title-banner` is accidentally hidden, because the linear gradient removed the background color. The above selector might be a way to target all of them at once (but I haven't tested that).

- http://wporg-news.test/page/2/
- http://wporg-news.test/?s=george
- http://wporg-news.test/2020/
- http://wporg-news.test/author/iandunn/
- http://wporg-news.test/tag/5-8/

Reply:
@MaggieCabrera is this an easy fix?

Reply:
yes, I think so, on it!

Reply:
ok so what I did is a bit hacky, I added a `:before` that hides the mask only when the menu is closed for those cases, that way the mask is still visible when the menu is open. It's more consistent with the rest of the site that way. Are these always blue? because I'm making that assumption here.

Reply:
I think that's a safe assumption, at least for now 👍🏻